.travis.yml

```diff
@@ -1,26 +1,31 @@
 env:
   global:
     - secure: >-
-        vdj0YPhLs8fIkFhnAmLXd+4gCgWAQjri+z5b1+nKyRCADjnXRfJugArwQqVSdqBX5Nj/tVeAVTluqt5+yqaZ5fxNfTTjN0iTSg8Z6TOmZp8Kr4x/TKjXZ2KMh0bDFZcDH/mgOwH35f2AeYyg/6ECst4YntjZ8U3vhlzuBB40wDJ/cAGcoHugXXLM+TtIktl78riesYu2+bMZ+Y554kmfeyQX/F3eeKFxdOUwagfA6AitL3h+e5NXFoCNTijqaM2gzEySsC4I58VbJL7ZuQPZBGrzubmFY+4ZVXQ6GKb4baohHmBgp6IkBGoxGv0sxx0dF3K048ADRgN4rivFbLmxGqEcWNvNpjlV+9TyetnyGlppDYzjumZMHKHC5MI9jCocZVZmBChlhjn9CO+iGyA8cty3jOCxNwwzni378Nt6Fc5k2kbJts9Cw7K70TS6CWHn6yBu4VhdMlpBcrBmbMEzcsWaCEEQBsS6b5je/Q7tiRMuPmBLv6WULRNwfmDDT/uYKNJFzwXQV/0KLebKmVXCd/wcaJh3pTleGrw0C/mGulD88VgSby9qb5+OJROZjjBUNoqzW1uFIe4whNu7mSgYGV0c07LZEQ4YLBxGHQsad6vkcFicxcnS7IxzHRlOeDqhJANLK08I34oNXqKyYhKrTbv6wkLIpcCiBcG6q4rHqNE=
+        TunBXHMpmjUfz/kU0bIWd6X6ph1H2gxItZWrTrarjUeJnjf1CG/3IoTKO88EIsesUT55S3UUk05UofSx+CLGriwtpu007YYVNV91L+XXo5ruo/IdeNBUtyKZBu4wFMM5iNnI2h7JznYx74VjHBBVvq5unLykwHMVnBD3uH1pg7Xuaa7V/6EV1ebWU/CGYBWkUszdPv6RJjdriM/WCTkGu0is9ypDonCK9GFCnWm6yHyZ6EV5CLr97mMHjbi06hUAuKsxpnvcKhnv6kJRo6W5fgY4rDOmCgyzydtLbMoQSl0ZIdJIec2jehbWfoCUCq9WXPcI6GXDyyIp9f48DghtdcJ7IWKYqP8eTLzbOVTLE+J53oy18Rey4lf8CWuLw05HdlSjzGaF1UvKudhJGhjCKlf0QrJfGIrDpwhVYvY6AKB8toxm7QuqutiV5tvpN08zR6ABrY2xEQ8lf6j8HMg/ZxjiQcBYOtv4bUHQz24o2SoLROAg113lW1xI9/sn7lMbpKuJVehfT9/M11eVP/Vn44VQsJ/2FXHMm5XkQfyk4+4nJJNprosPU4e+vxe+wq5l/BipizTCmSIahcz9MJQNlaIOEGKvGOJPx5qdXBMB5u9+6liYCq6/VdBSojExdvswYZsBSsj/iql3FUMQOQ/9pUL/v1NkiSKMbBL/Csd3GTY=
     - secure: >-
-        MQ96Q3uGP7BvGi9q/zZM1DUJqWMTsjazFXD66SW+heGAmoFhcMLwuTOKBJQrWQeEVNTkOE8nRRrmVzeB/DFHWDcDI77uNUUwNOqsn+g092PgxvDN+7ApuELN1ZCgJ/QdHbUZTn2p0paVtYA03pf3QNuoHU9b7cn2PIKeRiveehU8exVcQYoq0SYnDEbZoWJFq9WwTAnJEnDy5DWZIYrsX6375mZdd8Is0hMxT6UhCdiNtTXC789Uq+HtT/0CMuUR20ejvGUTmsQSb+HPBW0Ix1PtU/TneZ6zQCa2tv+UPIYDzWwc3AHYBRmKCoibZ8QwwyRv7mZ/NYOzOqXqpDBSfIr3b0cDYfz4YyOmt3CmDi88noJUzIwogJfQctaWbVqeREx6xnClKTG/vK1drqSKBMB94/3HW91YQRwq/RHJ0IYu7qpz4E7G0JKyMqZEsorP59DZjrBts9DOTY3VWNcbcHBfP6pT8URNJvr9fCa7RC+i94nRW85Z0WXnqe//VvKFSzZWf35yA+ekKJyreIJanKcqJg71o2Ivj6tYoBQNogbWmbfTP9G/lex8EHCRRvqh/Sczy0YDppfhKS0T4JaI9L9QqJX5hMUGbvoJMCsjNgvq3f3seGowCLBOn4VJrJh6LxS9vtfoAQIIkrPH5tNc7cgdnoBD3Mz/MPUFG7cV1KM=
+        N+F324OMnxvjYS6gnlWacaf6uiyDc0m2O3xARjHoso/nLWLxPkrYItGA2YyGzyAhL2Fta0E3QS/3hN53jvO1DwDXht6ser2r8bLGolFD4MKhlWKFYJV7Te6WRA21lyLty/mx9YDw+DQIM6b3pa2DtkKEzp1+ho9pqIKnxxj2WbykSd0QVXeZKx5ZLIEWil7Ue875B6DaBxynK8KmYYF0elOmjy+Gx+cchH092YHrybqM4UYO9Zo/CamZgGHxbSwp6tI/gj1tFw/6kgtxdmuVAPYrQWG7C1ephV9av9DxtHuDRcKC8MI5lnBOS/jcfaQEOQMnzPI8Fe9pFezPA5eyAeCIGoxIoEG83uNQ6yXGcaPQm/Wv22VdPLYRzsp0xUixT5H/1lnrf0Sc0jM2j+pG1LIGa9Kn7RSImYvtRzKWO6x4gwlwLnrxffli8c1pskumzhpcfPVsKtzkCSZLvoHA/872BiPhKdG3jqbbBIH0IJA02TSfMn1+XxmVqD5MEmOVd56roTgjJXKeJ1wGSE1wO28neWbb0MR7wDe7iqrtNMIGHhIhrxFZ2JdgJuCILpKNj1Q1vdoomIJ4qNCo7jtxS9fISvyu7rq5iLQbTZhRJsLsvgq9JurJ9XuLZG0R9mJfGtJv5djpb8TlJ87272YvLsU1JIc9i/Q6EbjmAeaIdXI=
 node_js: '9'
 addons:
   firefox: latest
   chrome: stable
 script:
   - xvfb-run polymer test
-  - |-
     >-
```

Comment:
The lone `>-` on this line still creates the `-` file. I think these should instead be:
`- >-`